scalikejdbc-syntax-support-macro/src/test/scala/foo/AutoSpec.scala

```diff
@@ -14,9 +14,10 @@ class AutoSpec extends FlatSpec with Matchers with DBSettings {
     def apply(r: ResultName[Organization])(rs: WrappedResultSet): Organization = autoConstruct(rs, r)
   }
 
-  case class Person(id: Long, name: String, organizationId: Option[Long], organization: Option[Organization] = None, gorupId: Long = 0)
+  case class Person(id: Long, name: String, organizationId: Option[Long], organization: Option[Organization] = None, groupId: Long = 0)
   object Person extends SQLSyntaxSupport[Person] {
     def apply(s: SyntaxProvider[Person])(rs: WrappedResultSet): Person = autoConstruct(rs, s, "organization")
+    override lazy val columns = useField[Person]("organization")
```

Comment:
oh, sorry I forgot this PR...

> This object will be located under scalikejdbc package. Is scalikejdbc.entityFields
which returns column names really reasonable?

I agree that the FQCN seemes strange, however I think this API is a kind of DSL.
So `autoConstruct` breaks standard naming convention(starts upper case).

back on topic, I like following
```scala
override lazy val columns = autoColumns[Person]("organization")
```